server/modules/organisation/model/find.js

```diff
@@ -16,4 +16,33 @@ const findOrganisation = async (id) => {
   return res.rows[0];
 };
 
-export { findOrganisation };
+const findHelpDetails = async (orgLink) => {
```

Comment:
can you rename all `orgLink`s to `uniqueSlug` please, so its all consistent

Reply:
done